mne_bids/tests/test_dig.py

```diff
@@ -36,6 +37,8 @@
 data_path = testing.data_path()
 raw_fname = op.join(data_path, 'MEG', 'sample',
                     'sample_audvis_trunc_raw.fif')
+trans = mne.read_trans(op.join(data_path, 'MEG', 'sample',
+                               'sample_audvis_trunc-trans.fif'))
```

Comment:
It's better not to `read_*` at the root level because this happens during test *collection*. Better to leave as `str` and use it below instead. I would make this `trans_fname` and create `trans` below

Reply:
How do you avoid the scale? Divide the `T1.header.get_ras2vox()` by 1000?